docs/victory-voronoi-container/docs.md

```diff
@@ -0,0 +1,132 @@
+# VictoryVoronoiContainer
+
+`VictoryVoronoiContainer` calculates a voronoi diagram based on the data of every child component it
+renders. The voronoi data is used to associate a mouse position with its nearest data point(s).
+When `VictoryVoronoiContainer` is added as the `containerComponent` of your chart, changes in mouse
+position will add and remove the `active` prop on appropriate elements in the components of the
+chart.
+
+
+```jsx
+<VictoryChart containerComponent={<VictoryVoronoiContainer/>}>
+  <VictoryLine data={data} />
+  <VictoryBar data={moreData}/>
+</VictoryChart>
+```
+
+## Props
+
+`VictoryVoronoiContainer` uses a superset of props used by [VictoryContainer].
+
+### dimension
+
+When the `dimension` prop is set, voronoi selection will only take the given dimension into account.
+For example, when `dimension` is set to "x", all data points matching a particular x mouse position
+will be activated regardless of y value. When this prop is not given, voronoi selection is
+determined by both x any y values.
+
+### radius
+
+When the `radius` prop is set, the voronoi areas associated with each data point will be no larger
```

Comment:
"voronoi areas" is pretty technical. I would have to Google that to know what it is. Maybe just saying it's the maximum distance the mouse can be from a point before it is highlighted? This goes back to the Voronoi vs. ClosestPoint language I mentioned above.